packages/palette/UPGRADE.md

```diff
@@ -0,0 +1,55 @@
+# v3 to v4
+
+v3.3 and above of the palette package contains the updated palette for the brand refresh. In v4, the old colors are removed entirely from this package.
+
+## Imports
+
+### JS
+
+The named export for the new palette has changed from `uiColors` to `palette`. You'll need to import the new colors like so:
+
+```js
+import { palette } from '@leafygreen-ui/palette';
+```
+
+### Less
+
+The name of the included `.less` variables file has changed from 'ui-colors.less' to `palette.less`. You'll need to import the new file like so:
+
+```less
+@import '<path to node_modules>/@leafygreen-ui/palette/dist/palette.less';
+```
+
+## Color Variables
+
+### JS
+
+The name and structure of the JS variables is largely unchanged. Here are the notable changes:
+
+- The `focus` variable has been removed. It's recommended to update usage of this color to `palette.blue.light1`.
+- The `palette.red.dark1` variable has been removed. It's recommended to update usage of this color to `palette.red.dark2`.
+- We've added a new `palette.green.light1` color.
+- We've added a new `palette.blue.dark2` color.
+- We've added a new `palette.red.light1` color.
+- We've added purple to the palette. You can find `dark3`, `dark2`, `base`, `light2`, and `light3` values for the color.
+
+### Less
+
+There's a number of notable changes to the Less variables that are available.
+
+- All variables have moved from a `@leafygreen__[color name]` format to a `@palette__[color name]` format.
```

Comment:
Should we prefix so it's `@lgpalette`? It looks gross but is a nice sandbox. But maybe too overtly defensive.

Reply:
I was trying to distance the new palette from being "leafygreen", since it's more broad than that (though yes, the package itself is leafygreen). Given the specific BEM-like namespacing of these colors, I was thinking the likelihood of collisions is extremely minute. Do you think that's incorrect?

Reply:
Oh that's fair. Also requires a consumer to import the LESS file. Yeah, leave as is!